code/modules/projectiles/ammo_datums.dm

```diff
@@ -2886,6 +2886,30 @@
 /datum/ammo/xeno/oppressor_tail/proc/remove_tail_overlay(mob/overlayed_mob, image/tail_image)
 	overlayed_mob.overlays -= tail_image
 
+/datum/ammo/xeno/facehugger
+	name = "thrown hugger"
+	icon_state = "facehugger_thrown"
+	flags_ammo_behavior = AMMO_XENO|AMMO_SKIPS_ALIENS|AMMO_STOPPED_BY_COVER|AMMO_IGNORE_ARMOR
+
+	max_range = 4
+	shell_speed = AMMO_SPEED_TIER_1
+
+/datum/ammo/xeno/facehugger/on_hit_mob(mob/target, obj/item/projectile/P)
```

Comment:
Fix single var usage in this proc. See: https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/STYLES.md#dont-use-abbreviations